src/darwin/Framework/CHIP/MTRDeviceControllerParameters.h

```diff
@@ -171,8 +171,11 @@ MTR_AVAILABLE(ios(18.2), macos(15.2), watchos(11.2), tvos(18.2))
  * @param uniqueIdentifier The unique id to assign to the controller.
  *
  */
+- (instancetype)initWithXPCConnectionBlock:(NSXPCConnection * (^)(void) )xpcConnectionBlock
+                          uniqueIdentifier:(NSUUID *)uniqueIdentifier MTR_NEWLY_AVAILABLE;
```

Comment:
Might as well just:
```suggestion
                          uniqueIdentifier:(NSUUID *)uniqueIdentifier MTR_AVAILABLE(ios(26.0), macos(26.0), watchos(26.0), tvos(26.0));
```
here, right?